cogs/pokemon.py

```diff
@@ -949,42 +949,57 @@ async def get_page(source, menu, pidx):
     @checks.has_started()
     @commands.guild_only()
     @commands.command(rest_is_raw=True)
-    async def evolve(self, ctx, *, pokemon: converters.PokemonConverter):
+    async def evolve(self, ctx, args: commands.Greedy[converters.PokemonConverter]):
         """Evolve a pokémon if it has reached the target level."""
 
-        if pokemon is None:
+        if not all(pokemon is not None for pokemon in args):
             return await ctx.send("Couldn't find that pokémon!")
 
         member = await self.bot.mongo.fetch_member_info(ctx.author)
         guild = await self.bot.mongo.fetch_guild(ctx.guild)
 
-        if (evo := pokemon.get_next_evolution(guild.is_day)) is None:
-            return await ctx.send("That pokémon can't be evolved!")
```

Comment:
Same here probably

Reply:
Unless I'm reading this wrong, I believe this is on line 973?